README.md

```diff
@@ -12,38 +12,138 @@ $ mkdir "$HOME/data_server/results" -p
+
+Create a `docker-compose.sh` script with:
 
 ```shell
-$ podman run \
+set -o allexport
+source .env
+
+
+data_server_img="localhost/snappy:2"
+net=snappy
+pgvol=pgvol
+db_name=pg_svc
+webserver_name=snap-web
+
+
+docker network create $net
+docker volume create $pgvol
+docker rm -f $db_name $webserver_name
+docker volume rm $pgvol
+
+
+docker run \
     --detach \
-    --name=snappy \
-    --net=host \
-    --env DATA_SERVER_PUBLIC_HOST=localhost \
-    --env DATA_SERVER_PORT=7070 \
-    --env DATA_SERVER_LOG_LVL=info \
-    --volume "$HOME/data_server/results:/data_server/app/results:z" \ 
-    quay.io/openshift-scale/snappy-data-server
-```
+    --env-file=.env \
+    --name=$db_name \
+    --net=$net \
+    --network-alias=$db_name \
+    --volume $pgvol:/var/lib/postgresql/data \
+    postgres:12.3-alpine
 
-## Example
 
-A simple script to show **snappy** being deployed locally and interacting with it's API.
+docker run \
+    --detach \
+    --env POSTGRES_PORT= \
+    --env POSTGRES_SERVER=$db_name \
+    --env-file=.env \
+    --net=$net \
+    --rm \
+    $data_server_img ./scripts/prestart
 
-```shell
-./localhost-example
+
+docker run \
+    --detach \
+    --env POSTGRES_PORT= \
+    --env POSTGRES_SERVER=$db_name \
+    --env-file=.env \
+    --name=$webserver_name \
+    --net=$net \
+    --publish ${DATA_SERVER_PORT}:${DATA_SERVER_PORT} \
+    --volume "$HOME/data_server/results:/data_server/app/app/results:z" \
+    $data_server_img
 ```
 
+### Podman Invocation
+
+**Podman invocation is currently not available.**
+
 ## Environment Variables
 
-### DATA_SERVER_PUBLIC_HOST
+Define a `.env` with these environment variables.
+
+### Data Server Configuration
+
+---
+
+#### DATA_SERVER_LOG_LVL
+Default: `info`  
+Data server log level. Current [Uvicorn server](https://www.uvicorn.org) **options:** *'critical', 'error', 'warning', 'info', 'debug', 'trace'.*
+
+#### DATA_SERVER_PORT
+Default: `7070`  
+This application's service port.
+
+#### DATA_SERVER_PUBLIC_HOST
 Default: `localhost`  
 URL to public host of data server.
 
-### DATA_SERVER_PORT
-Default: `7070`  
-Service port.
+#### DATA_SERVER_SECRET
+Secret to encode passwords in database.
 
-### DATA_SERVER_LOG_LVL
-Default: `info`  
-Data server log level. Current [Uvicorn server](https://www.uvicorn.org) **options:** *'critical', 'error', 'warning', 'info', 'debug', 'trace'.
+#### POSTGRES_PORT
+Docker: Empty string  (i.e. `POSTGRES_PORT=`)  
+Podman: `5432`  
+Postgres service port. 
+
+#### POSTGRES_SERVER
+Docker: name of the database network alias, see `docker-compose.sh`.  
+Podman: `localhost`  
+Domain name, or IP address, hosting the postgres service.
+
+### Postgres Service Configuration
+
+---
+
+#### POSTGRES_PASSWORD
+Postgresql database super user password.
+
+### Initial Users Configuration
+
+---
+
+#### FIRST_SUPERUSER
+Username for the first super user.
+
+#### FIRST_SUPERUSER_PASSWORD
+Password for the first super user.
+
+#### FIRST_TEAMUSER
```

Comment:
can we just have `FIRST_SUPERUSER*` and remove `FIRST_TEAMUSER*`
@mfleader

Reply:
sure can!